plugins/woocommerce-admin/client/payments-welcome/strings.tsx

```diff
@@ -378,4 +378,59 @@ export default {
 
 		getInTouch: __( 'Get in touch', 'woocommerce' ),
 	},
+	apms: {
+		addMoreWaysToPay: __(
+			'Add more ways for buyers to pay',
+			'woocommerce'
+		),
+		seeMore: __( 'See more', 'woocommerce' ),
+		paypal: {
+			title: __( 'PayPal Payments', 'woocommerce' ),
+			description: __(
+				'Enable PayPal Payments alongside WooCommerce Payments. Give your customers another way to pay safely and conveniently via PayPal, PayLater, and Venmo.',
+				'woocommerce'
+			),
+		},
+		amazonpay: {
+			title: __( 'Amazon Pay', 'woocommerce' ),
+			description: __(
+				'Enable Amazon Pay alongside WooCommerce Payments and give buyers the ability to pay via AmazonPay. Transactions take place via Amazon embedded widgets, so the buyer never leaves your site.',
```

Comment:
Should this say `Amazon Pay`, not `AmazonPay`? It looks like that's how it is referred to elsewhere in the plugin and on Amazon's website. 🙂

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/34581/commits/eabb148f3b86602d3ec84ecf16735b79d86a0382.